Src/Particle/AMReX_Particle.H

```diff
@@ -424,16 +424,19 @@ std::ostream&
 operator<< (std::ostream& os, const Particle<NReal, NInt>& p)
 {
     os << p.id()   << ' '
-       << p.cpu()  << ' ';
+       << p.cpu()  << "\n";
```

Comment:
Could you revert the changes to this file? I think this might cause some of the regression tests for some codes that look at ASCII to fail.